### ISSUE TITLE
Disappearing water bucket from dispenser!

### ISSUE THREAD

**Issue by mcbigboy** [mezz/JustEnoughItems#734]
I know what you are thinking, what could this have to do with your mod and that's a good question. Here are the steps I took to come to this conclusion.  I cleaned out all my mods and loading a vanilla Minecraft and started a new world.  I then set up  a dispenser with a water bucket inside it and triggered a redstone signal. The water came out as it should have. I applied the signal again to remove the water. This worked as it should. I did this three or four more times to make sure that all was good.

I closed out Minecraft and restarted this time with Forge and did the same steps as before. It worked fine. Repeating the restart method I added IC2 my biggest mod next and tested each mod until I found the mod that the water bucket disappeared. After loading JEI I ran the test as before but this time there was no water after the second time. Looking inside the dispenser the bucket was gone. This was the problem I was having in my world that started me looking into this. 

I closed down and removed JEI and started up again and reset the test. Once again the test was working as before.  I'm not just sure what else I could tell you. 

If you want you can email me and I will send my cell number if you want to tell more about this. I don't do good with writing.  I've just started writing mod for 1.10 and having lots of fun. mcbigboy@gmail.com

**Comment by mezz** [comment]
Thanks for the report!
I can dispense water buckets with no issues, using Forge and JEI. 
I think you will have to try to narrow down your search more carefully before we can find the cause.

**Comment by mcbigboy** [comment]
I will do the test again today. I know what mods I had load in. I will find which mods combo is the one causing the problem. Thanks for getting back with me.

**Comment by mcbigboy** [comment]
James, I feel bad I've been doing some kind of programming and trouble shooting for over 30 year. I've been disable for the last 10 year and gone down hill. They say your mind goes first.  I should have done more testing before I accused JEI of cause the problem.  After more testing I found that when I had JEI and Industrial Craft 2 Experimental for 1.10.2. I test all my other mods and only these two together cause the problem.  I then tested just IC2 and found that it was the culprit. 

I was lucky there was a new update for IC2 that fix the problem. I now have all my mods working. Add the water bucket problem is fixed too.  Thanks for you help.
